test/integration/model/create.test.js

```diff
@@ -956,7 +956,15 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       return UserNull.sync({ force: true }).then(() => {
         return UserNull.create({ username: 'foo2', smth: null }).catch(err => {
           expect(err).to.exist;
-          expect(err.get('smth')[0].path).to.equal('smth');
+
+          const smth1 = err.get('smth')[0] || {};
+
+          expect(smth1.path).to.equal('smth');
+          expect(smth1.originalType || smth1.type).to.match(/notNull Violation/);
```

Comment:
ah, right - the test actually works because there is no `originalType` but to be really correct it should read `smth1.type || smth1.origin` (to reflect what is going on inside the actual class)

I'll fix this too